src/core/Akka.TestKit/Akka.TestKit.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <EmbeddedResource Include="Configs\TestScheduler.conf;Internal\Reference.conf" />
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
```

Comment:
We should probably just take a dependency on FluentAssertions in the TestKit. I don't know why we haven't yet. 

I'll make a bigger issue for this and we'll revisit that prior to releasing v1.4.30.